test/unit/BPool.t.sol

```diff
@@ -1558,11 +1558,14 @@ contract BPool_Unit_ExitswapPoolAmountIn is BasePoolTest {
     bPool.exitswapPoolAmountIn(tokenOut, _fuzz.poolAmountIn, _minAmountOut);
   }
 
-  function test_Revert_NotFinalized() public {
+  function test_Revert_NotFinalized(
+    ExitswapPoolAmountIn_FuzzScenario memory _fuzz,
+    uint256 _minAmountOut
+  ) public happyPath(_fuzz) {
     _setFinalize(false);
 
     vm.expectRevert('ERR_NOT_FINALIZED');
-    bPool.exitswapPoolAmountIn(tokenOut, 0, 0);
+    bPool.exitswapPoolAmountIn(tokenOut, _fuzz.poolAmountIn, _minAmountOut);
```

Comment:
Could you fuzz all the function's general parameters within the `ExitswapPoolAmountIn_FuzzScenario` struct?